### PR TITLE
editorial: s/web page/developers

### LINKED ISSUE #437: Web page and developer
The spec sometimes talks about developers and sometimes about web pages (which are implying the developer). Should just talk about [developers... developers... developers](https://www.youtube.com/watch?v=KMU0tzLwhbE).

### PR DESCRIPTION
**Pull request by bentian**
Fixes #437

***
[Preview](https://s3.amazonaws.com/pr-preview/bentian/browser-payment-api/developers.html) | [Diff](https://s3.amazonaws.com/pr-preview/w3c/browser-payment-api/de90f44...bentian:cd70c72.html)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1867,16 +1867,16 @@
         </p>
         <p>
           After the payment request has been accepted and the
-          <a>PaymentResponse</a> returned to the page but before the page calls
-          <a>complete()</a> the payment request user interface remains in a
-          pending state. At this point the user interface ought not offer a
+          <a>PaymentResponse</a> returned to developers but before developers
```
Comment (marcoscaceres) [inline index.html]: this one should probably be phrased in terms of the caller or script.

File: index.html [hunk 2]
```diff
@@ -1867,16 +1867,16 @@
...
+          call <a>complete()</a> the payment request user interface remains in
+          a pending state. At this point the user interface ought not offer a
           cancel command because acceptance of the payment request has been
-          returned. However, if something goes wrong and the page never calls
+          returned. However, if something goes wrong and developers never call
```
Comment (marcoscaceres) [inline index.html]: this one needs to be singular.

File: index.html [hunk 3]
```diff
@@ -215,13 +215,13 @@
         };
       </pre>
       <p>
-        A web page creates a <a>PaymentRequest</a> to make a payment request.
+        Developers create a <a>PaymentRequest</a> to make a payment request.
```
Comment (marcoscaceres) [inline index.html]: Singular here.

File: index.html [hunk 4]
```diff
@@ -360,15 +360,15 @@
...
+              then <a>throw</a> a <a>TypeError</a>, optionally informing
+              developers that at least one <a>payment method</a> is required.
               </li>
               <li>For each <var>paymentMethod</var> of <var>methodData</var>:
                 <ol>
                   <li>If the length of the
                   <var>paymentMethod</var>.<a>supportedMethods</a> sequence is
                   zero, then <a>throw</a> a <a>TypeError</a>, optionally
-                  informing the developer that each <a>payment method</a> needs
+                  informing developers that each <a>payment method</a> needs
```
Comment (marcoscaceres) [inline index.html]: singular here. Check the others. Might make more sense as just "the developer"

File: index.html [hunk 5]
```diff
@@ -324,7 +324,7 @@
           </pre>
           <p>
             The <var>options</var> object contains information about what
-            options the web page wishes to use from the payment request system.
+            options developers wish to use from the payment request system.
```
Comment (marcoscaceres) [inline index.html]: Maybe: "...information the developer needs from the end-user to perform the payment (e.g., the person's name and shipping address)."

File: index.html [hunk 6]
```diff
@@ -360,15 +360,15 @@
           <li>Process payment methods:
             <ol data-link-for="PaymentMethodData">
               <li>If the length of the <var>methodData</var> sequence is zero,
-              then <a>throw</a> a <a>TypeError</a>, optionally informing the
-              developer that at least one <a>payment method</a> is required.
+              then <a>throw</a> a <a>TypeError</a>, optionally informing
+              developers that at least one <a>payment method</a> is required.
```
Comment (marcoscaceres) [inline index.html]: singular here.

File: index.html [hunk 7]
```diff
@@ -394,14 +394,14 @@
...
+              optionally informing developers that the value is invalid.
               </li>
               <li>If the first character of
               <var>details</var>.<a>total</a>.<a data-lt=
               "PaymentItem.amount">amount</a>.<a data-lt=
               "PaymentCurrencyAmount.value">value</a> is U+002D HYPHEN-MINUS,
-              then <a>throw</a> a <a>TypeError</a>, optionally informing the
-              developer that the total can't be negative.
+              then <a>throw</a> a <a>TypeError</a>, optionally informing
+              developers that the total can't be negative.
```
Comment (marcoscaceres) [inline index.html]: same here..

File: index.html [hunk 8]
```diff
@@ -394,14 +394,14 @@
               "PaymentItem.amount">amount</a>.<a data-lt=
               "PaymentCurrencyAmount.value">value</a> is not a <a>valid decimal
               monetary value</a>, then <a>throw</a> a <a>TypeError</a>;
-              optionally informing the developer that the value is invalid.
+              optionally informing developers that the value is invalid.
```
Comment (marcoscaceres) [inline index.html]: same...

File: index.html [hunk 9]
```diff
@@ -597,7 +597,7 @@
         </h2>
         <div class="note">
           <p>
-            The <a>show()</a> method is called when the page wants to begin
+            The <a>show()</a> method is called when developers want to begin
```
Comment (marcoscaceres) [inline index.html]: "a developer"

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
Some of these need to be changed to singular: either "a developer" or "the developer", where it makes sense.

**Comment by dontcallmedom** [comment]
Marked as non-substantive for IPR from ash-nazg.

**Comment by dontcallmedom** [comment]
Marked as non-substantive for IPR from ash-nazg.

File: index.html [hunk 10]
```diff
@@ -1998,11 +1999,11 @@
           };
         </pre>
         <p>
-          The <a>PaymentRequestUpdateEvent</a> enables the web page to update
+          The <a>PaymentRequestUpdateEvent</a> enables developers to update
           the details of the payment request in response to a user interaction.
         </p>
         <p>
-          If the web page wishes to update the payment request then it should
+          If developers wish to update the payment request then they should
```
Comment (marcoscaceres) [inline index.html]: I feel like this one should be singular, but stick with gender neutral "they". Also, we shouldn't say "should" there... also, wish is kinda vague (I know it's not your wording, but let's fix it now :)). So:

> If a developer wants to update the payment request, then they need to

File: index.html [hunk 11]
```diff
@@ -709,13 +710,13 @@
         </h2>
         <div class="note">
           <p>
-            The <a>abort()</a> method is called if the web page wishes to tell
+            The <a>abort()</a> method is called if developers wish to tell
```
Comment (marcoscaceres) [inline index.html]: change this to singular, to match the note above.

**Review by marcoscaceres — CHANGES_REQUESTED** [review]
Nearly there.

File: index.html (hunk 10 above)
Comment (bentian) [reply index.html]: So there would be inconsistency between _the developer_ and _they_. Is this a convention for gender neutrality?

Comment (marcoscaceres) [reply index.html]: yes, that's a convention.

**Review by marcoscaceres — APPROVED** [review]
Lgtm. Thanks Ben!

**Pull request merged**